examples/ex25.cpp

```diff
@@ -389,27 +389,21 @@ int main(int argc, char *argv[])
    //     applying any necessary transformations such as: assembly, eliminating
    //     boundary conditions, applying conforming constraints for
    //     non-conforming AMR, etc.
-   a.Assemble();
+   a.Assemble(0);
 
    OperatorHandle Ah;
```

Comment:
Could be renamed to `OperatorHandle A`, now that the monolithic `SparseMatrix A` is not used anymore.